### PR TITLE
change fanchor to 0

### PR DESCRIPTION
**Pull request by daslyfe**
- kinda breaking change: will change patterns without the fanchor set using fenv to be a bit brighter
- It's time to make this change, it is a source of confusion when giving workshops why the filter env behaves in a bipolar fashion by default. Imagine strumming an instrument harder, and the harmonics fading faster. Bipolar envelopes can also cause clicks if the cutoff dips too low.

### CODE REVIEW

**Comment by felixroos** [comment]
Agreed that 0 is a better default!
It would be handy to have a quick way to repair old patterns. `fanchor` isn't even documented, so basically all patterns using a filter envelope will sound different. maybe there could be something like `setDefault('fanchor', 0.5)` . 
In superdough, all default values can then be set like `{ fanchor = getDefault('fanchor') }`

**Comment by daslyfe** [comment]
> Agreed that 0 is a better default! It would be handy to have a quick way to repair old patterns. `fanchor` isn't even documented, so basically all patterns using a filter envelope will sound different. maybe there could be something like `setDefault('fanchor', 0.5)` . In superdough, all default values can then be set like `{ fanchor = getDefault('fanchor') }`

Cool sounds good :) I added the setDefault function. Im not sure how to go through the db and modify existing patterns though to add the old default

**Comment by felixroos** [comment]
looks good, thanks for adding! I think the last thing missing is something like `resetDefaults`, which is called in `resetEditor`, so defaults from other patterns won't interfere when switching patterns. 
For the database, I can try to cook up an SQL query to add `setDefaullt?.('fanchor', 0.5)` to any pattern that includes then word `lpenv`

**Comment by daslyfe** [comment]
What if we had a version tag at the top of the editor? like // @ver 2.15 
That way we could have different defaults for different versions if needed and we would not need to update the DB

**Comment by felixroos** [comment]
continuing now here, because discord is too ephemeral: yep opt-out would also work with the version tag.. so if no version tag is present, use current defaults. if there is one, then use the defaults for that version.

**Comment by felixroos** [comment]
fixes https://github.com/tidalcycles/strudel/issues/1055

File: packages/superdough/superdough.mjs [hunk 1]
```diff
@@ -24,6 +24,52 @@ export function getSound(s) {
...
+  Object.keys(defaultsobj).forEach((key) => {
+    setDefaultValue(key, defaultsobj[key]);
+  });
+}
+export function resetDefaultValues() {
+  defaultControls = new Map(Object.entries(defaultDefaultValues));
+}
+export function setVersionDefaults(version) {
+  resetDefaultValues();
+  if (version === '1.0') {
```
[the remark this answers is not in the document]
Comment (daslyfe) [reply packages/superdough/superdough.mjs:68]: yes, this makes sense to me 💯

**Comment by daslyfe** [comment]
I'm guessing that you want to do the DB operation for adding the version tag before committing this?

**Comment by felixroos** [comment]
> I'm guessing that you want to do the DB operation for adding the version tag before committing this?

ok, merging now then quickly running the db update when it's deployed

**Comment by felixroos** [comment]
db fix is out, i've added // @version 1.0 to the bottom of each pattern in the db. personal patterns might still be off, so I guess we can mention that in the changelog for 1.1 that you can add the version tag to get the og version

**Pull request merged**